docs/source/miniconda.rst

```diff
@@ -1,102 +1,6 @@
-
-.. _quick-command-line-install:
-
-Quick command line install
-==========================
-
-These quick command line instructions will get you set up quickly with the latest Miniconda installer. For graphical installer (`.exe` and `.pkg`) and hash checking instructions, see :doc:`Installing Miniconda <miniconda-install>`.
-
-.. tab-set::
-
-   .. tab-item:: Windows
-
-      These three commands quickly and quietly install the latest 64-bit version of the installer and then clean up after themselves. To install a different version or architecture of Miniconda for Windows, change the name of the ``.exe`` installer in the ``curl`` command.
-
-      .. code-block:: none
-
-         curl https://repo.anaconda.com/miniconda/Miniconda3-latest-Windows-x86_64.exe -o miniconda.exe
-         start /wait "" miniconda.exe /S
-         del miniconda.exe
-
-      After installing, open the Anaconda Prompt (miniconda3) program to use Miniconda3. The following commands initialize conda for the cmd.exe and Powershell shells:
-
-      .. code-block:: none
-
-         conda init cmd.exe
-         conda init powershell
-
-   .. tab-item:: macOS
-
-      These four commands quickly and quietly install the latest M1 macOS version of the installer and then clean up after themselves. To install a different version or architecture of Miniconda for macOS, change the name of the ``.sh`` installer in the ``curl`` command.
-
-      .. code-block:: bash
-
-         mkdir -p ~/miniconda3
-         curl https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-arm64.sh -o ~/miniconda3/miniconda.sh
-         bash ~/miniconda3/miniconda.sh -b -u -p ~/miniconda3
-         rm -rf ~/miniconda3/miniconda.sh
-
-      After installing, initialize your newly-installed Miniconda. The following commands initialize for bash and zsh shells:
-
-      .. code-block:: bash
-
-         ~/miniconda3/bin/conda init bash
-         ~/miniconda3/bin/conda init zsh
-
-   .. tab-item:: Linux
-
-      These four commands quickly and quietly install the latest 64-bit version of the installer and then clean up after themselves. To install a different version or architecture of Miniconda for Linux, change the name of the ``.sh`` installer in the ``wget`` command.
-
-      .. code-block:: bash
-
-         mkdir -p ~/miniconda3
-         wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O ~/miniconda3/miniconda.sh
-         bash ~/miniconda3/miniconda.sh -b -u -p ~/miniconda3
-         rm -rf ~/miniconda3/miniconda.sh
-
-      After installing, initialize your newly-installed Miniconda. The following commands initialize for bash and zsh shells:
-
-      .. code-block:: bash
+.. raw:: html
 
-         ~/miniconda3/bin/conda init bash
-         ~/miniconda3/bin/conda init zsh
+        <html><head><meta http-equiv="refresh" content="0; URL='../../projects/miniconda/en/latest/'" /></head><body></body></html>
```

Comment:
Let's use sphinx-redirects for this, not a custom file, please, like in https://github.com/conda/conda/commit/bf050328b5cc930185fedaacda8c6229a32bc93e

Reply:
I went ahead and did the same for the conda and conda-build files, for consistency.